src/index.js

```diff
@@ -309,7 +308,7 @@ class ReactTooltip extends React.Component {
     let desiredPlace = e.currentTarget.getAttribute('data-place') || this.props.place || 'top'
     let effect = switchToSolid && 'solid' || this.getEffect(e.currentTarget)
     let offset = e.currentTarget.getAttribute('data-offset') || this.props.offset || {}
-    let result = getPosition(e, e.currentTarget, ReactDOM.findDOMNode(this), desiredPlace, desiredPlace, effect, offset)
+    let result = getPosition(e, e.currentTarget, this.tooltipRef, desiredPlace, desiredPlace, effect, offset)
```

Comment:
Is tooltipRef ever set?

Reply:
[tooltipRef is set here](https://github.com/wwayne/react-tooltip/blob/d53942a0c5ab88db828a781281cf0b9b94e2986e/src/index.js#L556)